### ISSUE TITLE
Include test runner version number in `MATCHING LINTERS` table

### ISSUE THREAD

**Issue by andrewvaughan** [oxsecurity/megalinter#3014]
**Is your feature request related to a problem? Please describe.**
Combing through debug logs to find the version number of a runner is frustrating when trying to determine is an issue with an old version of a linter (in my case, Vale) or my content.

**Describe the solution you'd like**
Include the version of the runner being used in the existing `MATCHING LINTERS` table for easy reference.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by echoix** [comment]
I agree that it could be a good idea! (Although the table might be already a bit large to fit in one line)

In the meantime, at least when you are debugging for a failing linter, the logs are in collapsible sections, and that info is right underneath the section at the top. So yah, it means scrolling back up.
